test/fuzz/fuzz_test_util.cpp

```diff
@@ -76,6 +76,7 @@ bool IsValid(spv_target_env env, const opt::IRContext* ir) {
   std::vector<uint32_t> binary;
   ir->module()->ToBinary(&binary, false);
   SpirvTools t(env);
+  t.SetMessageConsumer(kConsoleMessageConsumer);
```

Comment:
This seems fine but I just want to check it is intentional.

Reply:
Thanks - yes it is.